_docs/generators/context-map-generator.md

```diff
@@ -77,31 +77,30 @@ As a second example, the following Context Map represents our [insurance company
 }
 </pre></div>
 
-For this Context Map our generator produces the following graphical result:
+Our generator produces the following graphical result for the above Context Map:
 
 <a href="/img/context-map-generator-insurance-sample.png">![Insurance Company Example Context Map](/img/context-map-generator-insurance-sample.png)</a>
 
 ## User Guide
 The Context Map generator can be used within our Eclipse plugin as described below. 
 
 ### System Requirements
-The generator requires [Graphviz](https://www.graphviz.org/) to be installed on your system since it is based on it.
+The generator requires [Graphviz](https://www.graphviz.org/) to be installed on your system because it uses it behind the scenes:
 
  * Ensure [Graphviz](https://www.graphviz.org/) is installed on your machine.
- * Ensure the binaries of the [Graphviz](https://www.graphviz.org/) installation are part of your PATH environment variable and can be called from
-   the command line.
-    * Especially on Windows this is not the case after the installation of [Graphviz](https://www.graphviz.org/). The default installation path is
+ * Verify that the binaries of the [Graphviz](https://www.graphviz.org/) installation are part of your PATH environment variable and can be called from the command line, for instance by executing `dot -V` from the command line. <!--  on Window; same for Linux? --> 
```

Comment:
@socadk, yes, PATH variable exists on Linux as well. But on Linux (at least Ubuntu) the path of Graphviz is automatically added with the installation, as written in the sub-bullet. To be adjusted: remove comment or clarify instructions.